script/foreman-admin

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/env ruby
+# -*- encoding: utf-8 -*-
+
+require 'rubygems'
+require 'clamp'
```

Comment:
The question is not about using the installer or not. I can disable foreman_cli in the installer to have a Foreman server without hammer and everything will work just fine.